src/components/addItems/AddItemButton.tsx

```diff
@@ -76,10 +88,10 @@ function AddItemButton() {
   async function handleClick() {
     // Handle a request to add a list of items
```

Comment:
Let'd do some data validation here before submitting. Check that the email value is a valid email. Can look up the common regex for it. Also check it isn't empty. You can add a new error state that becomes true when the email is invalid and then makes the input field red.

Reply:
I gave this a try and did some testing. Now the submit button is disabled until valid items are in the drawer and the email passes the regex. The email input field is red until a valid email is entered. Please let me know if this is what we wanted.